### PR TITLE
PAPILIO-72 General Feeds

### PR DESCRIPTION
**Pull request by mth-1012**
## General info

An endpoint to fetch all activities available (unfiltered) from the DB. Pagination is implemented

## Task description

- Get details of an `Activity`
- Get all `Activity`s from DB, unfiltered and **unsorted** (needs update in the future)
- Paginate for smaller-sized response
- Return `totalPages` and `currentPage` for client to keep track
- Apply schemas

## Manual testing

- Pull from branch `PAPILIO-72-feeds`
- Remember to include `.env` and `credentials.json`
- Run `npm install`
- Run `npm run build`
- Run `npm start` to start the server
- Start testing with Postman using this endpoint
image

### How to use it

- Endpoint: `/api/activity/getFeeds?<optional-queries>`
- Optional queries: `page=<page-value>&size=<size-value>`
- DEFAULT: 
*- `page-value = 1` since first page, has to be >= 1
*- `size-value = 20` has to be >= 5

### CODE REVIEW

File: src/api/routes/activity-routes.ts [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+import { activityController } from '../controllers';
 
 const router = express.Router();
 
 /** GET */
 
-router.get('/activity/get/:activityId', (req, res) => {
-    res.send(`GET /activity/get/${req.params.activityId}`);
-});
+// FIXME: force activityId to be numerical, might consider using UUID in future
```
Comment (p-carrier) [inline src/api/routes/activity-routes.ts:8]: Have you open a JIRA task for this?

**Review by p-carrier — COMMENTED** [review]
The code seems right. Only 1 thing that you should keep track, I left a comment.

**Comment by p-carrier** [comment]
I tested it in Postman it it works

**Review by p-carrier — APPROVED** [review]
Good to go for more 🚀

File: src/api/routes/activity-routes.ts (hunk 1 above)
Comment (mth-1012) [reply src/api/routes/activity-routes.ts]: Fixed in this PR

**Review by p-carrier — APPROVED** [review]
It looks good to me. I tested it in postman and it works. I can't get the page when I request less than 5 items per page or the page 0 which is what we want. And there is no error when getting pages where there are no items.

🚀 that PR

**Comment by p-carrier** [comment]
fix the conflicts though

**Review by AC159 — APPROVED** [review]
All works when testing with postman and code looks correct

**Pull request merged**